pennylane/qnodes/decorator.py

```diff
@@ -178,20 +180,18 @@ def qnode(device, *, interface="autograd", mutable=True, diff_method="best", pro
             * ``"finite-diff"``: Uses numerical finite-differences.
 
             * ``None``: a non-differentiable QNode is returned.
+    Kwargs:
+        h (float): step size for parameter shift or the finite
+            difference method
 
-        properties (dict[str->Any]): additional keyword properties passed to the QNode
-    """
+        kwargs (dict[str->Any]): additional keyword arguments passed to the QNode
```

Comment:
I would remove this one for now (all individual keyword arguments should instead be listed here when needed, as you've done with `h` above :) )

Reply:
Sounds good, I'm removing this!

One thing here: would we still want to include the individual keyword arguments here (such as `h`)? As in the `BaseQNode` we would be omitting them, not sure if it would make sense to have them in the decorator.

Reply:
Definitely good to have them in the decorator! The decorator is the docstring that is user facing, it is very unlikely the user will be looking at the BaseQNode docstring :)